### DISCUSSION TITLE
【求助】头像怎么换

### DISCUSSION THREAD

**Discussion by JobNumber-9527** [share121/inter-knot#2470] [其他]
有人知道头像怎么换吗，我一换头像就提示我“真的出了点问题，我们无法处理那张照片。”咋办

**Comment by cpu644** [comment]
你好！这里是绳网技术支援团队。请确保你上传的图片小于1mb并小于500*500。

**Comment by cpu644** [comment]
为了稳定性建议使用.jpg或.png格式的图片。别的格式有可能出现兼容性错误。
